src/components/shared/static-map/static-map.tsx

```diff
@@ -1,9 +1,9 @@
-import { V2_MediaWidths } from "@lifesg/react-design-system/v2_media";
 import { useEffect, useRef, useState } from "react";
 import { IColor } from "../../../services/onemap/types";
 import { TestHelper } from "../../../utils";
 import { LocationHelper } from "../../fields/location-field/location-helper";
 import { StaticMapElement, StaticMapWrapper, staticMapDimensions } from "./static-map.styles";
+import { Breakpoint } from "@lifesg/react-design-system";
```

Comment:
use the subpath imports

```suggestion
import { Breakpoint } from "@lifesg/react-design-system/theme";
```